apps/site/components/Common/Searchbox/ChatInteractions/index.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+
+import { ArrowDownIcon } from '@heroicons/react/24/solid';
+import type { Interaction } from '@orama/core';
+import { ChatInteractions } from '@orama/ui/components';
+import { useScrollableContainer } from '@orama/ui/hooks/useScrollableContainer';
+import { useTranslations } from 'next-intl';
+
+import { ChatMessage } from '../ChatMessage';
+import styles from './index.module.css';
```

Comment:
tiny nit: we always keep styles after all other imports and separated by an empty line (afaics)